### DISCUSSION TITLE
Is it possible to add custom args to the run command ?

### DISCUSSION THREAD

**Discussion by samir-** [geeksblabla/blanat#30] [Q&A] [answered]
if not, can we have in the readme the evaluation process. 
i saw there is a rust judge, but it it prone to be changed over time to include custom args that may change contestants code execution time ?

**Comment by aboullaite** [comment, marked as answer]
We might add more runtimes, but no custom arguments to ensure fairness. This might change in future editions of the challenge, but not for this one

**Reply by samir-** [reply]
so ,  we may have different possible JDKs for instance ?  is it upon asking for it or left to the judge ?

**Reply by aboullaite** [reply]
Unfortunately not. Same runtime for same lang. check this discussion https://github.com/geeksblabla/blanat/discussions/25
